package.json

```diff
@@ -10,9 +10,16 @@
     "server": "json-server --watch ./server/database.json",
     "lint": "ng lint"
   },
+  "lint-staged": {
+    "src/**/*.{js,ts,scss,md,html,json}": [
+      "prettier --write",
+      "git add"
+    ]
+  },
   "husky": {
     "hooks": {
-      "pre-commit": "lint"
+      "pre-commit": "ng lint && lint-staged",
```

Comment:
`lint-staged` is specified which is also omitted from `devDependencies` of `package.json` file
you must use the `--save-dev` flag